galaxy_ml/utils.py

```diff
@@ -602,6 +602,12 @@ def get_scoring(scoring_json):
     if scoring_json.get('secondary_scoring', None) not in (None, 'None', '')\
             and scoring_json['secondary_scoring'] !=\
             scoring_json['primary_scoring']:
+        # secondary_scoring used to be a comma separated string.
+        # At some point it turned into a list. Adding logic here
+        # to support both list and string (for backward compatability)
+        if type(scoring_json['secondary_scoring']) is list:
+            scoring_json['secondary_scoring'] =\
+                ",".join(scoring_json['secondary_scoring'])
         scoring = set(scoring_json['secondary_scoring'].split(','))
```

Comment:
@kxk302  Thanks for making this.
How about this way?
```suggestion
        scoring = scoring_json['secondary_scoring']
        if not isinstance(scoring, list):
            scoring = scoring.split(',')
        scoring = set(scoring)
```

Reply:
This is good too.